easybuild/easyblocks/i/ifort.py

```diff
@@ -56,6 +57,8 @@ def sanity_check_step(self):
             elif LooseVersion(self.version) >= LooseVersion("2013_sp1"):
                 binprefix = "bin"
                 libprefix = "lib/intel64/lib"
+                if LooseVersion(self.version) >= LooseVersion("2016"):
+                    libprefix = "lib/intel64_lin/lib"
```

Comment:
stick to `intel64` (a symlink to `intel64_lin`), since they may change `intel64_lin` *again* in the future...